week_1/Project/youtube-fe/styles/header.css

```diff
@@ -0,0 +1,123 @@
+.header {
+    height: 55px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
```

Comment:
@codal-kcolaco what is justify-content? how many values it has?

Reply:
@abhi-codal  justify-content is used to align flex items along main-axis. It has around 14 values among which are **center**, **space-between**, **flex-start** and many more.